From 2a4bfd158597ea41f5037c8544c69e26e43ba2a1 Mon Sep 17 00:00:00 2001
From: Liang Chen <cl@rock-chips.com>
Date: Wed, 8 Jul 2020 15:40:01 +0800
Subject: [PATCH] ARM: config: rockchip: enable CONFIG_ARM_CPU_TOPOLOGY for
 rk3308 aarch32

We need enable CONFIG_ARM_CPU_TOPOLOGY for scheduler.
The default value of CONFIG_ARM_CPU_TOPOLOGY is y, so just delete
'# CONFIG_ARM_CPU_TOPOLOGY is not set'.

Change-Id: I6917cfe7d521fa00885e82743738f12f832512f9
Signed-off-by: Liang Chen <cl@rock-chips.com>
---
 arch/arm/configs/rk3308_linux_aarch32_debug_defconfig | 1 -
 arch/arm/configs/rk3308_linux_aarch32_defconfig       | 1 -
 2 files changed, 2 deletions(-)

diff --git a/arch/arm/configs/rk3308_linux_aarch32_debug_defconfig b/arch/arm/configs/rk3308_linux_aarch32_debug_defconfig
index 7f87ef50d214..8058b7001d58 100644
--- a/arch/arm/configs/rk3308_linux_aarch32_debug_defconfig
+++ b/arch/arm/configs/rk3308_linux_aarch32_debug_defconfig
@@ -30,7 +30,6 @@ CONFIG_ARCH_ROCKCHIP=y
 # CONFIG_CACHE_L2X0 is not set
 # CONFIG_ARM_ERRATA_643719 is not set
 CONFIG_SMP=y
-# CONFIG_ARM_CPU_TOPOLOGY is not set
 CONFIG_ARM_PSCI=y
 CONFIG_PREEMPT=y
 CONFIG_HZ_1000=y
diff --git a/arch/arm/configs/rk3308_linux_aarch32_defconfig b/arch/arm/configs/rk3308_linux_aarch32_defconfig
index 966a2c7a5a60..be7a24585b76 100644
--- a/arch/arm/configs/rk3308_linux_aarch32_defconfig
+++ b/arch/arm/configs/rk3308_linux_aarch32_defconfig
@@ -30,7 +30,6 @@ CONFIG_ARCH_ROCKCHIP=y
 # CONFIG_CACHE_L2X0 is not set
 # CONFIG_ARM_ERRATA_643719 is not set
 CONFIG_SMP=y
-# CONFIG_ARM_CPU_TOPOLOGY is not set
 CONFIG_ARM_PSCI=y
 CONFIG_PREEMPT=y
 CONFIG_HZ_1000=y
-- 
2.35.3

